src/sql/sqlops.proposed.d.ts

```diff
@@ -703,6 +708,10 @@ declare module 'sqlops' {
 
 	export interface InputBoxComponent extends Component, InputBoxProperties {
 		onTextChanged: vscode.Event<any>;
+		/**
+		 * Event that's fired whenever enter is pressed within the input box
+		 */
+		onInputEntered: vscode.Event<string>;
```

Comment:
onEnterKeyPressed

Reply:
Yeah - I was trying to keep it generic in case we want to update this later on to include other things besides enter. I guess since I named the other one stopEnterPropagation though that'd already be an issue so I'll change this